docs/data/destinations/braze.md

```diff
@@ -31,10 +30,61 @@ To configure an Event Streaming integration from Amplitude to Braze, you need th
 - `app_id`: ID for the Braze app that should receive Amplitude events. Find this in your Braze Developer Console under **Settings**. See the [Braze documentation](https://www.braze.com/docs/api/identifier_types/#the-app-identifier-api-key) for more help.
 - `api_key`: The REST API Key used for authentication. Find this in your Braze Developer Console under **Settings**. See the [Braze documentation](https://www.braze.com/docs/api/basics/#rest-api-key) for more help.
 
-### Amplitude setup 
+### Amplitude setup
 
-1. In Amplitude, navigate to **Data Destinations**, then find **Braze - Event Stream**.
-2. Enter a sync name, then click **Create Sync**.
-3. Click **Edit**, then paste your Braze REST API Endpoint, REST API Key, and Braze App Identifier API Key.
-4. Use the _Send events_ filter to select the events to send. You can send all events, but Amplitude recommends choosing the most important ones.
-5. When finished, enable the destination and save.
+1. In Amplitude, navigate to **Data**.
+2. On the sidebar, under **Connections**, select **Catalog**.
+3. Select the **Destinations** tab.
+4. Select **Event Streaming**.
+5. Click **Braze - Event Streaming**.
+6. Enter a sync name, then click **Create Sync**.
+
+After you create the destination, you must configure the settings.
+
+#### Configure settings
+
+1. On the **Settings** tab, click **Edit**.
+2. Select your **Braze REST API Endpoint**.
+3. Enter your **REST API Key**.
+4. Enter your **Braze App Identifier API Key**.
+5. **Create & Update Users** creates users in Braze and update the properties of existing users when an Amplitude Identify API call is made.
+      1. To create and update users, toggle Create & Update Users to Enabled.
+      2. To select user properties to send, expand the Specify user properties to send panel, and select properties to forward. If you don't select any properties here, Amplitude doesn't include any.
```

Comment:
Throughout the docs here, these points are missing the styling (bolding). See the GA4 docs.